lessons/Early-Math/absolute-value-and-opposites-2/langs/en-us/pages/examples.adoc

```diff
@@ -0,0 +1,98 @@
+= Programming with Absolute Value and Opposite
+
+
+++++
+<style>
+#content td {padding: 0.5rem 5px !important}
+#content table .autonum::after { content: ')'; }
+#content th { text-align: center !important; }
+</style>
+++++
+
+== Predict
+
+@vspace{1ex}
+
+With your partner, predict if each equation on the table below is _true_ when `m` is defined as 6, 12, and 0. Note: an equation might be true for some values and false for others!
```

Comment:
You need to tell them what to do with their predictions! Add a sentence about placing a check in the boxes that will pass. (And make sure you do the same for all similar pages)

Reply:
struggled with wording here. Feel free to change if it's no good!